src/pages/workspaces/List.js

```diff
@@ -143,9 +144,14 @@ export const WorkspaceList = () => {
     }),
     initialFiltered), [accessLevelsFilter, filter, initialFiltered, projectsFilter, submissionsFilter, tagsFilter])
 
+  const { profile: { starredWorkspaces } } = useStore(authStore)
+  const starredWorkspaceIds = _.isEmpty(starredWorkspaces) ? [] : _.split(',', starredWorkspaces)
+
+  //Starred workspaces are always floated to the top
```

Comment:
It's unfortunate that this makes workspaces change place in the list when they are starred/unstarred. But I also think this sort order makes sense and I don't see a good way around that.

Reply:
I think it's pretty confusing/unexpected that it disappears when you toggle the star. I feel like what I've seen in other UIs is that the item you have toggled remains where it is until you refresh the page or select a sort option in the table.

Reply:
And I think that would be easy to do by removing the `onUpdate` call to refresh the workspaces.

Could you touch base with UX about the experience?

Reply:
It is unfortunately a bit more convoluted than just removing the onUpdate call. A bunch of state and sorting code needs to be updated as well. I am checking with UX about this now

Reply:
@MatthewBemis any update?

Reply:
Not yet, but I think we will have this resolved with Joy soon. I should have put this back in draft in the meantime- I'll do that now.

Reply:
@cahrens the conclusion with Joy is that we will keep the behavior as-is, but we will of course be listening carefully for user feedback and adjust the behavior if needed